pkg/apis/acme/v1alpha2/types_issuer.go

```diff
@@ -46,6 +51,26 @@ type ACMEIssuer struct {
 	Solvers []ACMEChallengeSolver `json:"solvers,omitempty"`
 }
 
+// ACMEExternalAcccountBinding is a reference to a CA external account of the ACME
+// server.
+type ACMEExternalAccountBinding struct {
+	// keyID is the ID of the CA key that the External Account is bound to.
+	KeyID string `json:"keyID"`
```

Comment:
KeyID can be potentially sensitive information. We should er on the side of caution and make this a secret reference

Reply:
I think at this point we can keep it as `keyID` - it's effectively the 'username' parameter here, and the 'secret' value should be sufficient to protect user accounts. It's not really explored in the RFC, and if needed in future, we can add an additional field for a secret ref variant of this, for those who *are* more sensitive about key IDs.